#4 implement car prototype manager service

ZsoltToth opened the pull request.

ZsoltToth committed:
cars - #4
service pom has been updated to version 1.1-SNAPSHOT
core dependency was increased to 1.1-SNAPSHOT

ZsoltToth committed:
cars - #4
CarPrototypeManagerServiceImpl.java has been added.
CarPrototypeDAO.java has been defined
CarDAO has been extended with search by plate number method.

judit0310 commented on cars/service/src/main/java/hu/uni/miskolc/iit/webdevelopment/examples/cars/dao/exceptions/PrototypeIDAlreadyExistsException.java:
```diff
@@ -0,0 +1,7 @@
+ * Created by tothzs on 2017.11.24..
+ */
+public class PrototypeIDAlreadyExistsException extends Exception {
+}
```
Missing constructors

ZsoltToth replied:
constructor has been added

judit0310 reviewed (changes requested):
Some issue point is not fulfilled.

ZsoltToth committed:
cars - #4
CarPrototypeManagerServiceImpl.java has been tested
CarPrototypeDAO.java is mocked
java compiler version has bee nset to 1.8

ZsoltToth reviewed (commented):
request is fulfilled
